### ISSUE TITLE
[release-0.4.5] "sample Custom Sitrep" is shown on game start.

### ISSUE THREAD

**Issue by Cjkjvfnby** [freeorion/freeorion#253]
I suggest to change text or remove it from release branch.

```
CUSTOM_SITREP_INTRODUCTION
This is a sample Custom Sitrep, which is enabled by %tech% and all the busy empire servants reporting information back to %system%
```

![sample_custom](https://cloud.githubusercontent.com/assets/171597/8917642/f605af32-34bb-11e5-8023-ca18cae65cc5.png)

**Comment by MatGB** [comment]
Why? I'm not sure on the exact wording, but it's a nice new feature which we want to encourage people to use, definitely need some links to the empire intelligence agency there to ensure people know how to use it.

**Comment by Cjkjvfnby** [comment]
`This is a sample Custom Sitrep`  looks like something temporary.

By the way haw about replacing all `SitRep` to `Situation report`?

**Comment by Dilvish-fo** [comment]
> or remove it from release branch

I'm just going to reinforce what Mat has said, the main point of this SitRep is to introduce this ability to FO newcomers (or to people only familiar with 0.4.4 'stable').  The article it links to also nicely fits in a primer on dismissing SitReps.  If you somehow see this SitRep as a detriment you'll really have to explain yourself.

One related thing I'll note, I realized I need to do a little bugfix on the 'undefended natives' sitrep.   It's currently sending the sitrep even if the natives are stealthed, which would not be terrible by itself since this info is already found in the Pedia, _but_ this sitrep is only sent if they are undefended, which you would not normally know if they are stealthed.  So I'm going to exclude stealthed natives.

**Comment by Dilvish-fo** [comment]
> By the way haw about replacing all SitRep to Situation report? 

Ah, yes, I agree; I had actually been meaning to do that but it kept slipping my mind.

> `This is a sample Custom Sitrep`  looks like something temporary.  

I don't really understand why this conveys 'temporary' to you.  It's a sample because it is pre-provided, not actually customized by the player.   Although 'sample' and 'tempoprary' may sometimes go together, the latter does not imply the former.   Please go ahead and suggest language that you think would read better.  We can doctor up the English, just focus on what ideas you think should be conveyed in this message.

**Comment by Cjkjvfnby** [comment]
`This is a sample Custom Sitrep` looks like you add  new feature and share it with other developers. And it will gone then there will be other usages of this feature.

Maybe:
`Welcome to situation report center. It is enabled by %tech%. Here you can find all information provided by empire agents.` 

IMHO help button near `filters` is more suitable to show info from `Empire Intelligence agency`

PS. This sitrep named `Custom 1` in Filters, can you rename it to `Welcome`?
PPS. Don't forget to add empty line after citation.

**Comment by Dilvish-fo** [comment]
(existing): "This is a sample Custom Sitrep, which is enabled..."

> Welcome to situation report center. It is enabled by

I think I really want the message to refer to the message itself, not the overall SitRep panel (which is what 'situation report center' really sounds like a reference to.)

Howabout, `"This is a customizable Situation Report, which is enabled...`

> IMHO help button near filters is more suitable to show info from Empire Intelligence agency

That's something that could be discussed for the future; the time for such 0.4.5 feature changes is past.  But even for the regular master line, my initial reaction is 'hmmm, nope.'  The whole point of these is to be a clear reminder of things you want to be nudged about.  Having to hit some button to see them really lessens that tremendously.  They can be controlled with the filters, and with dismissals, easily enough.

> PS. This sitrep named Custom 1 in Filters, can you rename it to Welcome?

Hmm, I had it as Custom 1 as part of its 'example' role, but I guess we do now have others templates there that server that role.  Changing this does make sense to me-- anyone else have an opinion?

**Comment by Cjkjvfnby** [comment]
As developer I like this SitRep.  As player, just see some wired stuff. (I guess that 90% of persons who would modify sitereps will uses developer version). Message about controls of this menu is ok for release. Word `custom`, `exampe`,  `sample` is not.

**Comment by Dilvish-fo** [comment]
> Word custom, exampe, sample is not.

You have some strong opinions, and some of those strong opinions, or at least what I understand of them, seem quite odd to me.  I would disagree about all of these, but "sample" and "example" don't matter much now, I think, since I am preferring "customizable".  However, it really sounds like you are saying, in response to my proposal of `"This is a customizable Situation Report` that you think using the word "customizable" is unacceptable.  Am I understanding you correctly?  If I am, could you explain a little about your understanding of the word "customizable" that makes you think it is unacceptable here?

**Comment by Cjkjvfnby** [comment]
It is customizable, You just need to study new programming language. Windows players with game installed to `Program Files` will not like your instructions.  As simple player I say: It is not customizable.

**Comment by Dilvish-fo** [comment]
> It is customizable, You just need to study new programming language.   

Ah, ~~I have learned quite a fair few in my life, hehe.  Any one in particular you are thinking I need to pick up now to understand you?~~  OK, Geoff's follow-up made me realize what you meant here.   I've responded in the post below his.

> Windows players with game installed to Program Files will not like your instructions. As simple player I say: It is not customizable.

"It is not customizable" because what, you are somehow unable to change the resource directory?  If the player needed to twiddle with settings a bit on Windows in order to edit the default content, then I could understand an argument that it is not _easily_ customizable; but even then it would be customizable.  And it is far simpler than that since the player can just change the content directory to one they have edit rights to.   Is that really your point-- that FO content is not really editable because it is too much of a nuisance for Windows users?

**Comment by Dilvish-fo** [comment]
I'll acknowledge that the existing instructions on modifying the SitReps might be incomplete for Windows folks.  So one of those people (hey, you're one!) should add a Pedia article on modifying FO content on Windows, and this article could link to that one.  I don't think any of that has to do with the message itself, though.

**Comment by geoffthemedio** [comment]
I'm feeling like feature is prematurely enabled...

The issue with editing the script files in the default directory is an important concern. It might be better to locate such a file in the user directory instead.

Calling the sitreps customizable when it requires learning a new scripting language is, I agree, a bit misleading.

Unless I'm mistaken, these sitreps are also not clearly explained as local-only, and not having any effect on a remote server.

They also seem to allow players to circumvent all visibility / information rules. It might be better if they were generated / run only on the local client's known gamestate, not on the server like any normal tech-derived sitrep.

I also don't like the example custom sitreps having english text written directly into them, and being enabled by default for any language.

While they may be quite useful for an experienced coder / testing, I'm no so sure they work well for new players now.

**Comment by Dilvish-fo** [comment]
> Unless I'm mistaken, these sitreps are also not clearly explained as local-only, and not having any effect on a remote server.

I haven't tested it, but I'm pretty sure you are mistaken on that.  The SitReps are generated by the server just like any other SitRep.  All the players qualify to receive the same SitReps (unless the server owner intentionally edits them to exclude others, and that could apply to a lot of our content, and is a different kind of issue.)

> Calling the sitreps customizable when it requires learning a new scripting language is, I agree, a bit misleading.

Ah, that was what Cj meant about learning a new programming language.  Well, that could be another reason to not say _easily_, yes.  But you don't have to have a GUI interface to call something customizable. As I understand it though. the whole point of creating FOCS though was that be reasonable easy for a non-programmer to learn, so they could customize things.  That's nothing unique to these SitReps.   That was also part of the point of providing several initial templates, in a way these customizable SitReps are a good intro primer to working with FOCS. 

> They also seem to allow players to circumvent all visibility / information rules.

This doesn't do anything to change the extent that is 'allowed', but by suggesting the messages be customized it does make it easier to inadvertently bypass visibility, as I did with the initial Natives message.

> It might be better if they were generated / run only on the local client's known gamestate, not on the server like any normal tech-derived sitrep.

Yes, I agree, that would be better.

> I also don't like the example custom sitreps having english text written directly into them, and being enabled by default for any language.

Hmm, yes, that is why (at Ouaz' prompting) I changed the intro one to use the UserStrings.  We could simply comment out the others and simply leave them as examples, if everyone thinks that would be best. Even just these intro ones are so very helpful though, I guess I'd be inclined to go ahead and make at least one more of them translated, so the player can really get an idea of just how helpful they can be.

> While they may be quite useful for an experienced coder / testing, I'm no so sure they work well for new players now.

I don't see anyway the pre-existing ones are useful only for us and not also for new players.  The one about Gas Giant Generators is probably even _more_ helpful for a newcomer than for an experienced player.

**Comment by geoffthemedio** [comment]
By server only, I meant that players making a modification to their local custom sitreps will see them in single player games, but not in games connected to a remote server. Ideally they would be based on the local client's universe, and would work for locally or remotely served games.

**Comment by Deepsloth** [comment]
I agree with Cjkjvfnby that the wording should be changed. "Sample Custom Sitrep" sounds technical and breaks the immersion into the game (which is very important for a first time player).

When i try a new roguelike, i don't like to be immediately welcomed with "btw you can write your own macros for your pet". That's what i want to know after playing for hours and deciding to keep it (even if i still won't do it though).

**Comment by Dilvish-fo** [comment]
Everyone who has expressed a concern here, please review the PR I just submitted and comment in it if you think someone more needs to be addressed for 0.4.5.  (I submitted it as a PR to master, but expect to cherrypick the final version to 0.4.5).
